rules/scala/private/provider.bzl

```diff
@@ -15,53 +15,61 @@ load(
 )
 
 def configure_bootstrap_scala_implementation(ctx):
-    return [
-        _ScalaConfiguration(
-            compiler_classpath = ctx.attr.compiler_classpath,
-            global_plugins = ctx.attr.global_plugins,
-            global_scalacopts = ctx.attr.global_scalacopts,
-            runtime_classpath = ctx.attr.runtime_classpath,
-            semanticdb_bundle = ctx.attr.semanticdb_bundle,
-            use_ijar = ctx.attr.use_ijar,
-            version = ctx.attr.version,
-        ),
-        _ScalaRulePhase(
-            phases = [
-                ("=", "compile", "compile", _phase_bootstrap_compile),
-            ],
-        ),
-    ]
+    scala_configuration = _ScalaConfiguration(
+        compiler_classpath = ctx.attr.compiler_classpath,
+        global_plugins = ctx.attr.global_plugins,
+        global_scalacopts = ctx.attr.global_scalacopts,
+        runtime_classpath = ctx.attr.runtime_classpath,
+        semanticdb_bundle = ctx.attr.semanticdb_bundle,
+        use_ijar = ctx.attr.use_ijar,
+        version = ctx.attr.version,
+    )
+    return struct(
```

Comment:
Huh, so I don't want to speak for Jaden, who actually knows what he's doing, but I seem to recall him saying that we needed it so we could get access to it from the aspect in bazel-bsp/hirschgarten. I did just update it to use `["_ScalaConfiguration"]` instead of `.scala_configuration` though, and afaict, it seems to work exactly the same.

Reply:
Ah ok, I think I fixed it so we won't need this. https://github.com/JetBrains/hirschgarten/commit/7077bff0467eca7bf20fc700edb6fce600cdfbd3